### ISSUE TITLE
关于文本识别finetune过程出现的问题

### ISSUE THREAD

**Issue by phb-shiyige-fw** [PaddlePaddle/PaddleOCR#9560]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Win10
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：ch_PP-OCRv3_rec_distillation.yml
- 运行指令/Command Code：tools/train.py -c configs/rec/PP_OCRv3/ch_PP-OCRv3_rec_distillation.yml
- 完整报错/Complete Error Message：

我尝试使用ch_PP-OCRv3_rec_distillation.yml进行文本识别的finetune，但是在训练的过程中发现acc和loss的波动非常大，呈现震荡的趋势；并且acc会出现在几个固定值反复的情况，并不是稳步上升的情况，具体如下图所示：
![微信截图_20230328100858](https://user-images.githubusercontent.com/94897119/228109515-d15bc2b2-661f-4062-b264-7502aaa6b2d6.png)
请问是什么地方出现错误了吗？我应该如何改进？附上完整的配置文件：

Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: ./output/rec_ppocr_v3_distillation2
  save_epoch_step: 100
  eval_batch_step: [1000, 2000]
  cal_metric_during_train: true
  pretrained_model: C:/Users/PHB/Desktop/ocr_table/PaddleOCR-dygraph/pretrain_models/ch_PP-OCRv3_rec_train/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: true
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 50
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_distillation.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Piecewise
    decay_epochs : [700, 800]
    values : [0.00003, 0.000003]
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 0

Architecture:
  model_type: &model_type "rec"
  name: DistillationModel
  algorithm: Distillation
  Models:
    Teacher:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: SVTR
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
        last_conv_stride: [1, 2]
        last_pool_type: avg
      Head:
        name: MultiHead
        head_list:
          - CTCHead:
              Neck:
                name: svtr
                dims: 64
                depth: 2
                hidden_dims: 120
                use_guide: True
              Head:
                fc_decay: 0.00001
          - SARHead:
              enc_dim: 512
              max_text_length: *max_text_length
    Student:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: SVTR
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
        last_conv_stride: [1, 2]
        last_pool_type: avg
      Head:
        name: MultiHead
        head_list:
          - CTCHead:
              Neck:
                name: svtr
                dims: 64
                depth: 2
                hidden_dims: 120
                use_guide: True
              Head:
                fc_decay: 0.00001
          - SARHead:
              enc_dim: 512
              max_text_length: *max_text_length
Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationDMLLoss:
      weight: 1.0
      act: "softmax"
      use_log: true
      model_name_pairs:
      - ["Student", "Teacher"]
      key: head_out
      multi_head: True
      dis_head: ctc
      name: dml_ctc
  - DistillationDMLLoss:
      weight: 0.5
      act: "softmax"
      use_log: true
      model_name_pairs:
      - ["Student", "Teacher"]
      key: head_out
      multi_head: True
      dis_head: sar
      name: dml_sar
  - DistillationDistanceLoss:
      weight: 1.0
      mode: "l2"
      model_name_pairs:
      - ["Student", "Teacher"]
      key: backbone_out
  - DistillationCTCLoss:
      weight: 1.0
      model_name_list: ["Student", "Teacher"]
      key: head_out
      multi_head: True
  - DistillationSARLoss:
      weight: 1.0
      model_name_list: ["Student", "Teacher"]
      key: head_out
      multi_head: True

PostProcess:
  name: DistillationCTCLabelDecode
  model_name: ["Student", "Teacher"]
  key: head_out
  multi_head: True

Metric:
  name: DistillationMetric
  base_metric_name: RecMetric
  main_indicator: acc
  key: "Student"
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: C:/Users/PHB/Desktop/rec_train/
    ext_op_transform_idx: 1
    label_file_list:
    - C:/Users/PHB/Desktop/rec_train/crop_img/rec_gt.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
#    - RecConAug:
#        prob: 0.5
#        ext_data_num: 2
#        image_shape: [48, 320, 3]
#        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 8
    drop_last: true
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: C:/Users/PHB/Desktop/rec_test/
    label_file_list:
    - C:/Users/PHB/Desktop/rec_test/crop_img/rec_gt.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 8
    num_workers: 4

**Comment by phb-shiyige-fw** [comment]
- RecConAug:
prob: 0.5
ext_data_num: 2
image_shape: [48, 320, 3]
max_text_length: *max_text_length
补充：配置文件中的这个部分被我注释掉了

**Comment by andyjpaddle** [comment]
前面是在warmup阶段，可以再训训看，另外epoch数和lr_decay注意要匹配
